README.md

```diff
@@ -43,7 +43,13 @@ This project uses the [pyrex](https://github.com/garmin/pyrex) tool to transpare
     docker images build-nilrt  # should print the image you just built
     ```
 
-4. #### Set up build environment
+1. #### Set `ni-main` feed version
+   If you're building a pre-release branch (`nilrt/master/hardknott`), set `NILRT_MAIN_FEED_VERSION` in `nilrt/sources/meta-nilrt/conf/distro/nilrt.inc` to last released ni-main feed.
```

Comment:
The `nilrt/master/hardknott` ref is going to become obsolete once we rebase. And I think builders should be overwriting the default variable value in their `local.conf`, rather than modifying the meta-nilrt layer. What about...

"""
If you are building on a `nilrt/master/*` branch ref (rather than a release branch) **and** if you are building outside of the NI corporate network, you will need to set the version of the `ni-main` opkg feed to one which has already been published to [`download.ni.com`](http://download.ni.com/#ni-linux-rt/feeds/). Do this by setting the `NILRT_MAIN_FEED_VERSION` bitbake variable to the latest published release. eg.
```bash
echo 'NILRT_MAIN_FEED_VERSION = "2022Q3"' >>./conf/local.conf
```
"""

And then we should probably put this task into the "Set up the build environment" step.

Reply:
Done